Source/DafnyLanguageServer/Handlers/Custom/VerificationHandler.cs

```diff
@@ -37,8 +37,12 @@ public class VerificationHandler : IJsonRpcRequestHandler<VerificationParams, bo
 
     var translatedDocument = await documentManager.CompilationManager.TranslatedDocument;
     var requestPosition = request.Position;
-    return GetTasksAtPosition(translatedDocument, requestPosition).
-      Any(taskToRun => documentManager.CompilationManager.Verify(translatedDocument, taskToRun));
+    var someTasksAreRunning = false;
+    var tasksAtPosition = GetTasksAtPosition(translatedDocument, requestPosition);
+    foreach (var taskToRun in tasksAtPosition) {
+      someTasksAreRunning |= documentManager.CompilationManager.Verify(translatedDocument, taskToRun);
```

Comment:
Interesting use of bitwise or to ensure the right-hand side is always evaluated!

Reply:
😅 I didn't even realise there's another risk of lazy evaluation here, but at least the test confirms this works.